apps/interpreter/package.json

```diff
@@ -18,5 +17,6 @@
   "publishConfig": {
     "access": "public",
     "registry": "https://registry.npmjs.org"
-  }
+  },
+  "version": "will be overridden by the root version"
```

Comment:
Interesting idea for documentation :)
You sure it works?

Reply:
Is inconsistent with other apps where the field was just removed.

Reply:
Maybe something like `the version field here is required for the interpreter, but the value will be overridden by the root version` would be better?

Reply:
Aaah! Makes sense, thx! :)

Reply:
You're welcome! Would it be better that way?

Reply:
Would turn it around, but yes :)